### PR TITLE
Additional annotations for PVC

### PR DESCRIPTION
**Pull request by ibrokethecloud**
Harvester UI, leverages additional annotations for the UI to identify Volumes / Images associated with a VM.

This PR introduces the same.

Related to: https://github.com/harvester/harvester/issues/2864

### CODE REVIEW

File: pkg/controllers/migration/virtualmachine.go [hunk 1]
```diff
@@ -647,3 +655,24 @@ func (h *virtualMachineHandler) tidyUpObjects(vm *migration.VirtualMachineImport
 	}
 	return nil
 }
+
+// generateAnnotations will generate the harvester reference annotations: github.com/harvester/harvester/pkg/ref
+func generateAnnotations(vm *migration.VirtualMachineImport, vmi *harvesterv1beta1.VirtualMachineImage) (map[string]string, error) {
```
Comment (guangbochen) [inline pkg/controllers/migration/virtualmachine.go:660]: can we try to reuse https://github.com/harvester/harvester/blob/master/pkg/controller/master/virtualmachine/pvc_util.go#L54-L62 if appliable?

Comment (guangbochen) [reply pkg/controllers/migration/virtualmachine.go:660]: let's leave it as a future enhancement and consider making `harvester/harvester` func more flexible to adapt, thanks.

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Review by bk201 — APPROVED** [review]
LGTM!

**Pull request merged**